source/partials/next_release/_release.md.erb

```diff
@@ -1,102 +1,54 @@
-* <%= link_to_commit 'ba495470c154cf46aadf3ac45c67544735391c61', 'Disable job settings link for pipelines defined in Config Repo on elastic profiles page.', 5604 %>
-* <%= link_to_commit '3f2fc49422a9fb93ccb6b938dfd8ea4ed75c92a2', 'Made status report button state on elastic profiles page consistent with the  status report button on plugins page ', 5776 %>
-* <%= link_to_commit '6c4da91079e3ea11e905da14f7cf709604e7d7a5', 'Fixed environments api version 2 etag generation issue.', 5807 %>
-* <%= link_to_commit '38e68d0ca9dfce367dd1d9a380e824595cd87d34', 'Render console log containing new line character ("\n").', 5037 %>
-* <%= link_to_issue 5705, 'Fix for high CPU usage on console logs/tests page.' %>
-* <%= link_to_pull_request 5897, 'Killing agent via stop command honors the process ID' %>
-* <%= link_to_issue 5788, 'Fixed back button navigation on dashboard.' %>
+* <%= link_to_issue 5956, 'Fixed issue with environments API incorrectly rendering encrypted variables as `******` instead of the correct encrypted value.' %>
+* <%= link_to_issue 5886, 'Do not allow assigning an agent to an environment defined in config repository environment.' %>
+* <%= link_to_issue 5968, 'Fix spurious `e.map is not a function` javascript error for notifications.' %>
+* <%= link_to_pull_request 5974, "Unable to expand/collapse environments with name containing special characters." %>
+* <%= link_to_issue 5962, "GoCD shows HTTP Basic Auth dialog when after user is logged out." %>
+* <%= link_to_pull_request 6011, "Fix errors with parsing git log when coloring is turned on using `.gitconfig`." %>
+* <%= link_to_pull_request 5967, "Environments api should accept both `encrypted_value` and `value`." %>
 
 <h4>New Deprecations</h4>
-* The Fedora 28 based docker image for gocd agent has been deprecated and will not be built after June 2019 release of GoCD. This will be done as Fedora 28 will reach end-of-life in June 2019.
-* The Dashboard API v2 has been deprecated. This version of the API will be removed in a release scheduled for June 2019. Version 3 of the API is available, and users are encouraged to use it.
 
+* elastic extension v4
+* elastic agent profile v1 (breaking change)
 
 <h4>Deprecations</h4>
-* The elastic agent plugin extension version 3 has been deprecated. This version will be removed in a release scheduled for Mar 2019. Plugin developers should use version 4 of the elastic agent plugin api to allow adding support for job completion request. The docker, docker swarm, kubernetes and ecs plugins have been migrated to the new versions, and users should upgrade their plugins to the latest version to see these new features.
+
 * The notification plugin extension version 3 has been deprecated. This version will be removed in a release scheduled for Mar 2019. Plugin developers should use version 4 of the notification plugin api.
```

Comment:
We may have to think about a new timeline to remove this or we have to migrate all gocd supported notification plugins. /cc @arvindsv @maheshp 

The same goes for the analytics plugin.

Reply:
@kritika-singh3 Agreed. Let's not remove it this time.